synchronizer/l1_worker_test.go

```diff
@@ -9,7 +9,7 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
-func Test_Exploratory(t *testing.T) {
+func TestExploratoryWorker(t *testing.T) {
 	t.Skip("no real test, just exploratory")
```

Comment:
Why is it skipped?

Reply:
It's just exploratory, to be able to execute real request using etherman to check the results